pages/common/histexpand.md

```diff
@@ -0,0 +1,36 @@
+# history expansion
+
+> Reuse and expand the shell history in bash or zsh.
+> More information: <https://www.gnu.org/software/bash/manual/html_node/History-Interaction.html>.
+
+- Run the previous command:
+
+`!!`
+
+- Run the previous command as root:
+
+`sudo !!`
+
+- Run a command with the last argument of the previous command:
+
+`{{command}} !$`
+
+- Run a command with the first argument of the previous command:
+
+`{{command}} !^`
+
+- Run a command `n` lines back in the history:
```

Comment:
Here we're referring to a specific entry, so "the" would be the appropriate article.
```suggestion
- Run the command `n` lines back in the history:
```